Content.Server/GameObjects/Components/Chemistry/ReagentDispenserComponent.cs

```diff
@@ -36,6 +38,7 @@ public class ReagentDispenserComponent : SharedReagentDispenserComponent, IActiv
 #pragma warning disable 649
         [Dependency] private readonly IServerNotifyManager _notifyManager;
         [Dependency] private readonly ILocalizationManager _localizationManager;
+        [Dependency] private readonly IEntitySystemManager _entitySystemManager;
```

Comment:
delete this nephew

Reply:
> It might be worth looking at _all_ the uses of `IEntitySystemManager` as a dependency now that `EntitySystem.Get` exists, but that's for a future PR.

agreed

Reply:
weird. it placed my quote reply here.